src/group_theory/order_of_element.lean

```diff
@@ -149,7 +149,7 @@ lemma sum_card_order_of_eq_card_pow_eq_one {n : ℕ} (hn : 0 < n) :
   ((finset.range n.succ).filter (∣ n)).sum (λ m, (finset.univ.filter (λ a : α, order_of a = m)).card)
   = (finset.univ.filter (λ a : α, a ^ n = 1)).card :=
 calc ((finset.range n.succ).filter (∣ n)).sum (λ m, (finset.univ.filter (λ a : α, order_of a = m)).card)
-    = _ : (finset.card_bind (by simp [finset.ext]; cc)).symm
+    = _ : (finset.card_bind (by simp [finset.ext]; intros; apply finset.disjoint_filter; cc)).symm
```

Comment:
This non-terminal simp doesn't look very hygienic. Also, the pattern `by foo; bar` is nowadays deprecated in favour of `by { foo, bar }`.